src/wallet/api/wallet_manager.cpp

```diff
@@ -434,12 +434,14 @@ std::string WalletManagerImpl::resolveOpenAlias(const std::string &address, bool
     return addresses.front();
 }
 
-std::tuple<bool, std::string, std::string, std::string, std::string> WalletManager::checkUpdates(const std::string &software, const std::string &subdir)
+std::tuple<bool, std::string, std::string, std::string, std::string> WalletManager::checkUpdates(const std::string &software, std::string &subdir)
```

Comment:
Don't have a ref please, this is asking for trouble as the caller will likely not expect a change. Besides, it precludes calling with a constant.

Reply:
ofc. fixed.